topology/University of Utah/CHPC/CHPC Group.yaml

```diff
@@ -75,6 +75,25 @@ Resources:
         Description: Compute Element
         Details:
           hidden: false
+  SLATE_US_UUTAH_LONEPEAK:
+    Active: true
+    ContactLists:
+      Resource Report Contact:
+        Primary:
+          ID:  40a181cc3b7a43593188c9607310d35cb079bda9
+          Name: Mitchell Steinman
+      Site Contact:
+        Primary:
+          ID:  443c32443113f2e2dbf7dc77c0e5697d4f27f90a
+          Name: Irvin Allen
```

Comment:
Hi @Mansalu, I would think that the contact information should resemble previous entries. Could you replace the contents of `ContactLists:` with the following?

```
      Administrative Contact:
        Primary:
          ID:  40a181cc3b7a43593188c9607310d35cb079bda9
          Name: Mitchell Steinman
      Security Contact:
        Primary:
          ID:  40a181cc3b7a43593188c9607310d35cb079bda9
          Name: Mitchell Steinman
      Site Contact:
        Primary:
          ID:  ddfa10c8a8500e5c31818e7c1a063b3ce5ab6a8b
          Name: CHPC OSG Support
```